internal/dataplane/translator/translator_test.go

```diff
@@ -5025,7 +5025,8 @@ func mustNewTranslator(t *testing.T, storer store.Storer) *Translator {
 		FillIDs:                           true,
 		ReportConfiguredKubernetesObjects: true,
 		KongServiceFacade:                 true,
-	})
+	}, func() kong.AbstractSchemaService { return nil }, // TODO: return a fake schema service
```

Comment:
Do we want to solve the TODO in this PR or do we have an issue that tracks this?

Reply:
Done.